code/datums/components/bakeable.dm

```diff
@@ -90,11 +90,24 @@
 	baked_result.pixel_y = original_object.pixel_y
 	used_tray.AddToPlate(baked_result)
 
+	var/list/asomnia_hadders = list()
+	for(var/mob/smeller in get_hearers_in_view(DEFAULT_MESSAGE_RANGE, used_oven))
```

Comment:
```suggestion
	for(var/mob/smeller as anything in get_hearers_in_view(DEFAULT_MESSAGE_RANGE, used_oven))
```

Reply:
I do not believe `get_hearers_in_view` returns only mobs so be careful with this.

Reply:
why would it return not mobs though

Reply:
We have movables that "hear", like radios, right? I think they're included.

Reply:
I'd rather trust [MrMelbert](https://github.com/MrMelbert) and close it